script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -565,9 +568,10 @@ Did Akiha eat already?"
 [sha:e325d71c47950e352f11d91940687554122283f5]{
 -- Page 33, Offset 42063.
 -- そう返すだけで一日分の体力を使ったが、疲れたから死ぬ訳でもなし、そう気にする事でもない。
-//It's unclear exactly what そう返す is here
+// It's unclear exactly what そう返す is here
 Just asking that question took up my whole day's worth of energy,
  but I guess it's not that big a deal.
+// Flag for me to think about
```

Comment:
The 'about to kill me' here just feels off. In the current phrasing it sounds too 'immediate' instead of metaphorical. Suggest
> It's not like I'll die of exhaustion or anything.

or 
> It's not like I'm about to die of exhaustion or anything.

if the about to is important

Reply:
the first one seems fine